ipyparallel/client/asyncresult.py

```diff
@@ -600,6 +628,8 @@ def display_outputs(self, groupby="type"):
                 plots together, then all of the second plots, and so on.
         """
         self.wait_for_output()
+        if get_ipython() is not None:
+            clear_output(wait=True)
```

Comment:
Why this clear output? I don't think this should be inside display output (it would make it impossible to display two AR outputs in one cell, for instance)

Reply:
Right, will remove. I wanted to avoid printing stdout/stderr twice (once while streaming, and once at the end).
How would you recommend handling that?

Reply:
I think it probably makes the most sense to not redisplay output if we are streaming. Basically, if streaming don't redisplay output. Only display output if not already streaming. That means removing the `display_outputs()` call if we have been streaming at the end of `parallel_execute`

Reply:
Makes sense. 
The current callback only handles `msg_type == 'stream'`. Should I handle `display_data` and `execute_result` as well? `display_outputs` also has `display(r)`, not sure how to handle that.
Or add an option in `display_outputs` to skip stdout/stderr prints?

Reply:
Or maybe a new function that only displays the results. Just committed a change that does that. Let me know if that would work.